CHANGELOG-developer.next.asciidoc

```diff
@@ -102,3 +102,4 @@ The list below covers the major changes between 7.0.0-rc2 and master only.
 - Update Go version to 1.14.7. {pull}20508[20508]
 - Add packaging for docker image based on UBI minimal 8. {pull}20576[20576]
 - Make the mage binary used by the build process in the docker container to be statically compiled. {pull}20827[20827]
+- Add Pensando distributed firewall beat
```

Comment:
This should moved to the Added/Filebeat section in CHANGELOG.next.asciidoc.
```suggestion
- Add Pensando distributed firewall module. {pull}21063[21063]
```

Reply:
I don't see a CHANGELOG.next.asciidoc, did you mean just to update it in CHANGELOG-developer.next.asciidoc? I think it's already in the correct place, unless I am missing something.

Reply:
It's this file https://github.com/elastic/beats/blob/master/CHANGELOG.next.asciidoc.

Reply:
Thanks.  Made the change and committed.